lib/msf/core/rpc/v10/rpc_core.rb

```diff
@@ -15,6 +15,11 @@ def rpc_stop
     self.service.stop
   end
 
+  def rpc_getg(var)
```

Comment:
Because it uses ```framework.datastore``` and not a module specific datastore. In addition,```RPC_Core``` currently also only  supports ```setg``` and ```unsetg```.